src/net/sourceforge/kolmafia/textui/Parser.java

```diff
@@ -4570,10 +4664,53 @@ private void enforceSince(String revision, final Range directiveRange) {
     }
   }
 
+  public final void error(final String msg, final String... otherInfo) {
+    this.error(this.getCurrentPosition(), msg, otherInfo);
+  }
+
+  public final void error(final Position start, final String msg, final String... otherInfo) {
+    this.error(this.rangeToHere(start), msg, otherInfo);
+  }
+
+  public final void error(final Range range, final String msg, final String... otherInfo) {
+    this.error(this.makeLocation(range), msg, otherInfo);
+  }
+
+  public final void error(
+      final Range start, final Range end, final String msg, final String... otherInfo) {
+    this.error(Parser.mergeRanges(start, end), msg, otherInfo);
+  }
+
+  public final void error(final Location location, final String msg, final String... otherInfo) {
+    this.diagnostics.add(
+        new AshDiagnostic(
+            location != null ? location : this.makeZeroWidthLocation(), Error, msg, otherInfo));
+  }
+
   public final void warning(final String msg) {
     RequestLogger.printLine("WARNING: " + msg + " " + this.getLineAndFile());
   }
 
+  /*
```

Comment:
Delete instead of commenting out code.

Reply:
This isn't something to be deleted, it is just "not added" yet because it would conflict with the existing one

Reply:
... then add it when you're ready to delete the existing one?

Reply:
based on what's coming, I preferred to at least show that it was coming rather than fitting something unexpected

Reply:
(is it still ok if it stays there?)

Reply:
poke poke pokedy prod pokey?

Reply:
>poke poke pokedy prod pokey?

I see a PR marked as "Changes Requested" by another reviewer so I am going to ignore it until there is evidence the requested changes have been made.  So the ball is in your court.  Make and commit the changes or make it clear why you do not intend to do so.  If you think you have already done so then it was not clear to me.

Reply:
> I see a PR marked as "Changes Requested" by another reviewer so I am going to ignore it until there is evidence the requested changes have been made. So the ball is in your court. Make and commit the changes or make it clear why you do not intend to do so. If you think you have already done so then it was not clear to me.

There were 5 "conversations" from heeheehee.
I successfully addressed and resolved two of them.
Two others were questions/concerns which I responded to (didn't mark them as resolved in case there's a follow-up, but until then, I can assume they are)

This leaves only this one, in which I marked a reason to not address the change. Was it insufficient/unclear?

Reply:
>Was it insufficient/unclear?

Obviously or I would have responded differently.

Can you flag something (a conversation) as resolved?  If you can then it would help if you did so.  If you tell me No then I will look harder and resolve the items I am comfortable resolving.  Then I'll reassess status.

Reply:
I can mark them as resolved, yes. However, I prefer to not assume that a conversation is resolved when I'm not "resolving" it as the other person expected (i.e. answering why I would/should not)

Reply:
when doing this, I prefer getting a confirmation from the other party that the reason is satisfactory

Reply:
In that case the nudge needs to be more specific :-)

Reply:
@fredg1 Do not commit any commented code. Remove it and then you can resolve the conversation.

Also I don't know how many times I have to tell you but stop prodding people if they don't respond to you for periods of less than 24 hours or over weekens. Stop doing it.

Reply:
...noted.
For now, I'll guess I'll just comply and delete the comment, since the intent, which was to just make you (collectively) aware that this method was coming, was achieved either way...